lua/swenv/api.lua

```diff
@@ -111,7 +116,7 @@ end
 
 local get_pixi_base_path = function()
   local current_dir = vim.fn.getcwd()
-  local pixi_root = Path:new(current_dir) / '.pixi'
+  local pixi_root = Path:new(current_dir):joinpath('.pixi')
```

Comment:
I still don't understand, is this change needed?

Reply:
Sorry, that was a mistkae.